storage/rocksdb/mysql-test/rocksdb_sys_vars/r/rocksdb_wal_recovery_mode_basic.result

```diff
@@ -6,7 +6,7 @@ INSERT INTO invalid_values VALUES('\'aaa\'');
 SET @start_global_value = @@global.ROCKSDB_WAL_RECOVERY_MODE;
 SELECT @start_global_value;
 @start_global_value
-1
+2
```

Comment:
I'm ok with the change in value of this and ROCKSDB_STATS_LEVEL variables, but could a description/rational be included in the commit message.

Reply:
These changes originate from RocksDB, as it changes its way of interpreting integers. I will detail the changes in the commit message.